eventmesh-dashboard-console/src/main/resources/application-dev.yml

```diff
@@ -53,5 +53,10 @@ logging:
 mybatis:
   type-aliases-package: org.apache.eventmesh.dashboard.console.entity
 
+decoration:
+  constantConfig:
+    all: true
+  corsEnable: true
```

Comment:
Please leave some comments describing each config's meaning. Since there isn't any document others can find for reference.